run.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+echo "Welcome to Unity Catalog Guided Setup"
+echo "#######################################"
+
+
+confirm() {
+  echo "Please confirm before continuing (y/n)".
+  read -r confirmed
+  if [[ $confirmed == 'n' ]]; then
+    exit 2
+  fi
+}
+
+#Setup installers
+if [[ $OSTYPE == "linux-gnu"* ]]; then
+      echo "This tool will ensure that required dependencies are installed. Your permissions may be elevated via sudo during this process and you may be prompted, please do not run this command as root."
+      confirm;
+      if command -v yum &> /dev/null
+      then
+         sudo yum install -y wget unzip
+      fi
+      if command -v apt-get &> /dev/null
+      then
+        sudo apt-get install wget unzip
+      fi
+      if ! command -v terraform &> /dev/null
+      then
+        sudo wget https://releases.hashicorp.com/terraform/1.1.3/terraform_1.1.3_linux_amd64.zip
+        sudo unzip ./ terraform_0.12.2_linux_amd64.zip –d /usr/local/bin
+      fi
+elif [[ "$OSTYPE" == "darwin"* ]]; then
+      echo "This tool will ensure that required dependencies are installed. Your permissions may be elevated during this process and you may be prompted to enter your credentials or to confirm, please do not run this command as root."
+      confirm;
+      if ! command -v brew &> /dev/null
+      then
+          echo "brew is not installed, installing..."
+          -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+          brew update
+          brew install terraform
```

Comment:
I assume you need another `if ! command -v terraform &> /dev/null` check, otherwise terraform won't be installed if brew already exists

Reply:
added